server/src/main/java/com/objectcomputing/checkins/services/questions/QuestionController.java

```diff
@@ -51,43 +45,29 @@ public void setQuestionService(QuestionServices questionService) {
                     .headers(headers -> headers.location(location(newQuestion.getQuestionid())));
         }
     }
-    //  I know these are commented out - I added them before splitting the story and will
-    // need them in the future - please ignore for now
-//
-//    /**
-//     * Find and read a question or questions given its id or text.
-//     *
-//     * @param questionid
-//     * @param text
-//     * @return
-//     */
-//
-//    @Get("/{?questionid,text}")
-//    public List<Question> findByValue(@Nullable UUID questionid, @Nullable String text) {
-//
-//        List<Question> found = questionService.findByValue(questionid, text);
-//        return found;
-//
-//    }
-//
-//    /**
-//     * Update the pending status of a skill.
-//     * @param question
-//     * @return
-//     */
-//    @Put("/")
-//    public HttpResponse<?> update(@Body @Valid Question question) {
-//
-//        if(null != question.getQuestionid()) {
-//            Question updatedQuestion = questionService.update(question);
-//            return HttpResponse
-//                    .ok()
-//                    .headers(headers -> headers.location(location(updatedQuestion.getQuestionid())))
-//                    .body(updatedQuestion);
-//        }
-//
-//        return HttpResponse.badRequest();
-//    }
+
+    /**
+     * Update the text of a question.
+     * @param question
+     * @return
+     */
+    @Put("/")
+    public HttpResponse<?> update(@Body @Valid Question question) {
+
+        if(question.getQuestionid() != null) {
+            Question updatedQuestion = questionService.update(question);
+            if (updatedQuestion != null) {
+                return HttpResponse
+                        .ok()
+                        .headers(headers -> headers.location(location(updatedQuestion.getQuestionid())))
+                        .body(updatedQuestion);
+            } else {
+                return HttpResponse.badRequest();
```

Comment:
General comment, but I think we need a story for global error handling, and exceptions created we can all reuse in our code.